platform-sdk/swirlds-common/build.gradle.kts

```diff
@@ -14,12 +14,7 @@ tasks.withType<JavaCompile>().configureEach {
     )
 }
 
-mainModuleInfo {
-    annotationProcessor("com.swirlds.config.processor")
-
-    runtimeOnly("resource.loader")
-    runtimeOnly("com.sun.jna")
```

Comment:
These are not requires. They are already on the module path. Overlooked these in one of the larger cleanups we did months ago.